### PR TITLE
remove text pragmas and pseudo comments

### PR DESCRIPTION
**Pull request by larshp**
This removes pseudo comments `"#EC NOTEXT` and pragmas `##NO_TEXT` in implementation section of classes in order to reduce noise in the code.

Plus enable linting so these examples does not reappear in the code

#3555

### CODE REVIEW

**Comment by larshp** [comment]
theres something wrong in `src/ui/zcl_abapgit_gui_page_patch.clas.abap`

File: src/ui/zcl_abapgit_gui_page_patch.clas.abap [hunk 1]
```diff
@@ -1,798 +1,797 @@
...
+  ENDMETHOD.
+
+
+  METHOD add_menu_end.
+
+    io_menu->add( iv_txt = 'Stage'
+                  iv_act = c_actions-stage
+                  iv_id  = 'stage'
+                  iv_typ = zif_abapgit_html=>c_action_type-dummy
 ).
```
Comment (FreHu) [inline src/ui/zcl_abapgit_gui_page_patch.clas.abap:188]: there's some strange character here

Comment (larshp) [reply src/ui/zcl_abapgit_gui_page_patch.clas.abap:188]: yea, I'll have a look tomorrow, probably something wrong with the quickfix, plus guess the linter should also give a parser error

Comment (larshp) [reply src/ui/zcl_abapgit_gui_page_patch.clas.abap:188]: stupid carriage returns, logic for catching this early implemented in https://github.com/abaplint/abaplint/pull/1269/commits/52a7661a44b6ab92ab3229e9d6b1f3f7fd07f6cf

**Pull request merged**